packages/alchemy/src/provider/__tests__/light-account.test.ts

```diff
@@ -140,27 +140,11 @@ const givenConnectedProvider = ({
 }: {
   owner: SmartAccountSigner;
   chain: Chain;
-}) => {
-  const dummyEntryPointAddress =
-    "0x1234567890123456789012345678901234567890" as Address;
-
-  return new AlchemyProvider({
+}) =>
+  createLightAccountAlchemyProvider({
```

Comment:
can we add a test that shows the minimal amount of params needed? looking at this test it still feels like there's a lot of required params

is it possible to create a provider just by doing?
```ts
createLightAccountAlchemyProvider({
  chain,
  owner,
  apiKey, // or jwt
})
```

Reply:
so I use rpcUrl + jwt here b/c we test in unit that jwt works.

also I override accountAddress b/c the original test mocked the address to avoid the async calls.

e2e tests have less params, but think a more important place to show how few params are needed are the docs (which i'll write)